depositor_email_notification_job.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_literal: true
+
+# TODO
+# How do we collect all of the counts for the past month only?
+# Do we expect user.stats to return multiple UserStat objects. How does this work?
```

Comment:
Per Rob, we can expect multi UserStats, which get created for each day. We'd need to find them all by date for the past month and sum the totals to report the new ones. 

`>= '2024-04-01' AND < '2024-05-01'` => find everything for the month of April. 

```ruby
[#<UserStat:0x0000ffff85feb188
  id: 1,
  user_id: 1,
  date: Thu, 23 May 2024 00:00:00.000000000 UTC +00:00,
  file_views: 0,
  file_downloads: 0,
  created_at: Thu, 30 May 2024 23:32:22.429000000 UTC +00:00,
  updated_at: Thu, 30 May 2024 23:32:22.429000000 UTC +00:00,
  work_views: 0>,
 #<UserStat:0x0000ffff85feb048
  id: 2,
  user_id: 1,
  date: Fri, 24 May 2024 00:00:00.000000000 UTC +00:00,
  file_views: 0,
  file_downloads: 0,
  created_at: Thu, 30 May 2024 23:32:22.432511000 UTC +00:00,
  updated_at: Thu, 30 May 2024 23:32:22.432511000 UTC +00:00,
  work_views: 0>,
 #<UserStat:0x0000ffff85feaf08
  id: 3,
  user_id: 1,
  date: Sat, 25 May 2024 00:00:00.000000000 UTC +00:00,
  file_views: 0,
  file_downloads: 0,
  created_at: Thu, 30 May 2024 23:32:22.435427000 UTC +00:00,
  updated_at: Thu, 30 May 2024 23:32:22.435427000 UTC +00:00,
  work_views: 0>,
 #<UserStat:0x0000ffff85feadc8
  id: 4,
  user_id: 1,
  date: Sun, 26 May 2024 00:00:00.000000000 UTC +00:00,
  file_views: 0,
  file_downloads: 0,
  created_at: Thu, 30 May 2024 23:32:22.437734000 UTC +00:00,
  updated_at: Thu, 30 May 2024 23:32:22.437734000 UTC +00:00,
  work_views: 0>,
 #<UserStat:0x0000ffff85feac88
  id: 5,
  user_id: 1,
  date: Mon, 27 May 2024 00:00:00.000000000 UTC +00:00,
  file_views: 0,
  file_downloads: 0,
  created_at: Thu, 30 May 2024 23:32:22.439853000 UTC +00:00,
  updated_at: Thu, 30 May 2024 23:32:22.439853000 UTC +00:00,
  work_views: 0>,
 #<UserStat:0x0000ffff85feab48
  id: 6,
  user_id: 1,
  date: Tue, 28 May 2024 00:00:00.000000000 UTC +00:00,
  file_views: 0,
  file_downloads: 0,
  created_at: Thu, 30 May 2024 23:32:22.441642000 UTC +00:00,
  updated_at: Thu, 30 May 2024 23:32:22.441642000 UTC +00:00,
  work_views: 0>,
 #<UserStat:0x0000ffff85feaa08
  id: 7,
  user_id: 1,
  date: Wed, 29 May 2024 00:00:00.000000000 UTC +00:00,
  file_views: 0,
  file_downloads: 0,
  created_at: Thu, 30 May 2024 23:32:22.443533000 UTC +00:00,
  updated_at: Thu, 30 May 2024 23:32:22.443533000 UTC +00:00,
  work_views: 0>]
```